tools/release-scripts/release-1.5.sc

```diff
@@ -0,0 +1,177 @@
+  println(s"Updated changelog can be found here: ${changelogDir / "changelog.md"}")
+}
+
+@main
+def main(): Unit = {
+
+  println("creating a temp directory")
+  val tempDir = makeTmpDir()
+  println(s"temp directory created at $tempDir")
+
+
+  val marathonRepoName = "marathon"
+  val universeRepoName = "universe"
+  val dcosOssRepoName = "dcos"
+  val dcosEeRepoName = "dcos-ee"
+
+  implicit val context = BuildContext(
+    WorkingDirectory(tempDir),
+    MarathonContext(GitRepository("mesosphere", marathonRepoName), MarathonRepoDirectory(tempDir / marathonRepoName)),
+    UniverseContext(GitRepository("mesosphere", universeRepoName), UniverseRepoDirectory(tempDir / universeRepoName)),
+    DcosContext(GitRepository("dcos", dcosOssRepoName), DcosOssRepoDirectory(tempDir / dcosOssRepoName)),
+    DcosEeContext(GitRepository("mesosphere", dcosEeRepoName), DcosEeRepoDirectory(tempDir / dcosEeRepoName))
+  )
+
+  val reviewers = Seq("jeschkies", "ichernetsky", "alenkacz", "timcharper", "kensipe", "wavesoft", "zen-dog")
+
+  val credentials = tokens.loadUserCredentials()
+  println(s"Credentials are successfully loaded: $credentials")
+
+  val githubClient = new github.GithubClient(credentials.github)
+  val jenkinsClient = new jenkins.JenkinsClient(credentials.jenkins._1, credentials.jenkins._2)
+
+
+  checkoutMarathonRepo()
+
+
+  implicit val marathonRepoDirectory = context.marathon.directory.path
+
+
+  val latestReleasedVerison = {
+    val lastReleasedTag = %%('git, "tag", "-l", "--sort=version:refname")(marathonRepoDirectory).out.lines.filter(_.contains("v1.5")).last
+    SemVer(lastReleasedTag)
+  }
+
+  println(s"Latest released marathon 1.5 version is ${latestReleasedVerison.toReleaseString()}")
+
+  val nextMarathonVersion = latestReleasedVerison.nextBuildVersion
+
+  checkIfShouldStartRelease(latestReleasedVerison)
+
+
+  val changeLog = changelog.generateChangelog()
+  updateChangelog(changeLog)
+  println("Please review the generated changelog and make necessary changes. Once you're done, hit Enter.")
+  scala.io.StdIn.readLine()
+  println("Commiting changes")
+  val branchName = commitAndPushChangelog(nextMarathonVersion)
+  println("Creating a marathon PR")
+  val prNumber = githubClient.createMarathonChangelog15PR(nextMarathonVersion.toReleaseString(), branchName)
+  githubClient.requestReviews(prNumber, reviewers)
+  ui.showSpinner("Waiting for the PR to be merged") {
+    githubClient.waitUntilMerged(prNumber)
+  }
+  val sha = githubClient.getMergeCommitSha(prNumber).get
+
+  println(sha)
+
+  //todo: Implement running a jenkins job
+  //  val queueId = jenkinsClient.start15ReleasePipeline(sha, nextMarathonVersion.toTagString())
+  //  jenkinsClient.waitForTheJenkinsBuild(queueId)
+
+
+
+  println("Creating a new tag and pushing it")
+  createAndPushTag(nextMarathonVersion)
+  println("Creating new release")
+  githubClient.createReleaseFromTag(nextMarathonVersion.toTagString(), changeLog)
+
+  //todo: Build mom ee here
```

Comment:
I don't think this should be part of this script. The Jenkins job should take care of it.